src/app/dishdetail/dishdetail.component.ts

```diff
@@ -8,5 +7,5 @@ import { Dish } from '../shared/models/dish';
 })
 export class DishdetailComponent {
   @Input()
-  dish?: Dish;
+  dish?: IDish;
```

Comment:
why we haven't import `IDish` as `DIsh` previosly?

Reply:
We switched to type definition files